src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersResponse.java

```diff
@@ -31,7 +31,8 @@
     private List<String> roles = new ArrayList<String>();
 
     public ProfessionalUsersResponse(ProfessionalUser user) {
-        this.userIdentifier = user.getUserIdentifier();
+        this.userIdentifier = StringUtils.isEmpty(user.getUserIdentifier())
```

Comment:
Better to use Java Optionals e.g. Optional<UUID> userIdOpt = new Optional(user.getUserIdentifier);